tests/test_properties.py

```diff
@@ -29,3 +29,12 @@ class B(enum.Enum):
 
     with pytest.raises(AttributeError, match=r"B.b has no attribute 'val'"):
         B.b.val
+
+def test_getting_enum_property() -> None:
```

Comment:
```suggestion


def test_getting_enum_property() -> None:
```

I _think_ black will complain here?

Reply:
```suggestion

def test_can_introspect_property_attributes() -> None:
```

Let's name the test so that the aspect we're testing is clear.